### ISSUE TITLE
Unable to Modify Sidebar

### ISSUE THREAD

**Issue by iSanjayAchar** [creativetimofficial/material-dashboard-pro-angular#40]
I'm trying to modify the sidebar li part when it's active but I'm unable to do so. I tried modifying `_sidebar-and-main-panel.scss` and `_navbars.scss` but both were of no use 

Where can I exactly make the changes?

**Comment by alexandru-paduraru** [comment]
@iSanjayAchar thank you for using our product. Did you compile the SCSS using Koala as it is presented in the documentation at the end of the file?

Best,
Alex

**Comment by iSanjayAchar** [comment]
Nope I didn't use Koala, I'm using Angular CLI's out of the box SCSS compiler. Is it mandatory to use Koala?

**Comment by alexandru-paduraru** [comment]
@iSanjayAchar no, you can use any compiler, are you sure it compiles correctly your changes? Please delete anything that is in material-dashboard.scss and see if it's compiling to the right location, that means the dashboards should be broken after you test this. Then let us know which changes you made and didn't work.

Best,
Alex

**Comment by iSanjayAchar** [comment]
Yup @alexandru-paduraru it is compiling properly. I deleted the entire ` _sidebar-and-main-panel.scss` and it did break the entire side-bar. But then I'm really not sure how this is working. My requirement is simple. I need to resize the fonts and i need to modify the `.li:active` where should I exactly change it? 

Thanks for the help in advance :)

**Comment by SNAPEHR** [comment]
Hi,
Any updates on this. I have the same issue as well. I am trying to modify the background-color of the side bar and have followed the exact steps in your documentation including using Koala. But all I see is a white background color instead of the cyan that I wanted to use.
Thanks for the help.

**Comment by iSanjayAchar** [comment]
@dragosct10 Can you update us?

**Comment by iSanjayAchar** [comment]
@dragosct10 & @alexandru-paduraru - Guy's can we have an update it? Our development has stopped

**Comment by alexandru-paduraru** [comment]
@iSanjayAchar sorry for the late response. Here are some videos that we made about how to change the properties of the li elements and the active element:

1. Changing regular li and a: https://vimeo.com/227738857
2. Changing the active li: https://vimeo.com/227738839

@SNAPEHR We made a video on how to work with the sidebar colors: https://vimeo.com/227738691

Hope this will help.

Best,
Alex
